### PR TITLE
Task-76010-76012-Add pie chart for service pet usage and bar chart for monthly revenue

### PR DESCRIPTION
**Pull request by ghost**
## Related Tickets:
- [#76010](https://edu-redmine.sun-asterisk.vn/issues/76010)
- [#76012](https://edu-redmine.sun-asterisk.vn/issues/76012)

## WHAT this PR does?
- Add pie chart for service pet usage
- Add bar chart for monthly revenue

## Notes Impacted Areas:
- N/A.

## Attachments:
- N/A.

### CODE REVIEW

**Comment by ghost** [comment]
Ready

File: resources/lang/en/chart.php [hunk 1]
```diff
@@ -0,0 +1,8 @@
+<?php
+
+return [
+    'pie_title'   => 'Monthly Revenue Distribution',
+    'bar_title' => 'Service Usage Distribution',
+    'total_revenue' => 'Total Revenue',
+    'service_usage' => 'Service Usage',
+];
```
Comment (ghost) [inline resources/lang/en/chart.php]:
```suggestion
return [
    'pie_title' => 'Monthly Revenue Distribution',
    'bar_title' => 'Service Usage Distribution',
    'total_revenue' => 'Total Revenue',
    'service_usage' => 'Service Usage',
];
```

File: resources/lang/vi/chart.php [hunk 2]
```diff
@@ -0,0 +1,8 @@
+<?php
+
+return [
+    'pie_title'   => 'Doanh Thu Hàng Tháng',
+    'bar_title' => 'Lượt dùng dịch vụ',
+    'total_revenue' => 'Tổng doanh thu',
+    'service_usage' => 'Lượt dùng dịch vụ',
```
Comment (ghost) [inline resources/lang/vi/chart.php]:
```suggestion
    'pie_title' => 'Doanh Thu Hàng Tháng',
    'bar_title' => 'Lượt dùng dịch vụ',
    'total_revenue' => 'Tổng doanh thu',
    'service_usage' => 'Lượt dùng dịch vụ',
```

File: resources/views/dashboard/includes/bar-chart.blade.php [hunk 3]
```diff
@@ -0,0 +1,3 @@
+   <div class="col-span-6" style="height: 300px">
```
Comment (ghost) [inline resources/views/dashboard/includes/bar-chart.blade.php]: ko viet style vao blade template

**Comment by thainh-3454** [comment]
Ready

**Comment by ghost** [comment]
Ready

**Pull request merged**